test/awsE2ESolutionSetup.sh

```diff
@@ -3,14 +3,20 @@
 # Note: As this script will deploy an E2E solution in AWS, it assumes all the dependencies of the migration solution (e.g. aws cli, cdk cli,
```

Comment:
No action necessarily required, just a question for you - are you having trouble with maintaining/extending/debugging this script?  Have we considered migrating its functionality to something like Python, etc?

Reply:
I'm 1000% onboard with migrating this script to python, so much so that I let AI take a swing at it the other day to see how well it could do with it, and with some parameters it did a decent job and might be worth looking into as a reference